layouts/shortcodes/project-adopter-pages.html

```diff
@@ -1,6 +1,6 @@
 <ul>
 {{ range (.Site.GetPage "/project-adopters").Pages }}
 {{ $title:= .Title }}
-<li><a href="{{.RelPermalink }}">{{ $title }}</a></li>
+<li><a href="{{ .RelPermalink }}index.html">{{ $title }}</a></li>
```

Comment:
why add a "index.html" can make it redirect to the correct page on https://api.eclipse.org/adopters/ ? 🤔

Reply:
This a quick fix for now. We will look at this in the new year.